dashboard/README.md

```diff
@@ -0,0 +1,21 @@
+## Dashboard disposition demo
+
+To run the demo using earthly:
+
+Set up ROS 2 sources
+```
+earthly +sources
+```
+
+Add the [`process_sarif`](https://github.com/space-ros/process_sarif) project.
+```
+git clone https://github.com/space-ros/process_sarif spaceros_ws/src/process_sarif -b build-results-archive
+```
+
+Run a build and test
+
+```
+earthly +build-test +test
```

Comment:
Just curious: wouldn't this command be the same as "earthly +test"? Does it just come down to style and which command is more clear about the intent?

Reply:
And in fact, earthly wouldn't even let me specify both. I was hoping that implicit vs explicit targets would change some behavior but as I can't specify multiple targets that is a moot point. I've changed it to just `earthly +test`